src/main/java/pl/byczazagroda/trackexpensesappbackend/service/WalletServiceImpl.java

```diff
@@ -18,6 +22,18 @@ public class WalletServiceImpl implements WalletService {
     private final WalletRepository walletRepository;
     private final WalletModelMapper walletModelMapper;
 
+    @Override
+    @Transactional
+    public WalletDTO updateWallet(UpdateWalletDTO dto) throws ResourceNotFoundException {
+        Wallet wallet = walletRepository.findById(dto.id()).orElseThrow(() ->
+        {
```

Comment:
I think, that this bracket should be in the previous line

Reply:
Done.